.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+version = 3.8.1
+runner.dialect = scala213
+
+fileOverride {
+  "glob:**/scala-3/**" {
+    runner.dialect = scala3
+
+    rewrite {
+      scala3 {
+        convertToNewSyntax = true
+        removeOptionalBraces = true
+      }
+    }
```

Comment:
Most of this configuration (if not all) won't be applied as we will write in Scala 2 and get that source code compiled in Scala 3 too

Reply:
Is there a way to do the opposite? Write in Scala 3 and have it compiled to Scala 2?

Reply:
Not that I'm aware of. It's Scala 3 that has many elements of retrocompatibility, not the other way around. We might need to specialise some code for the two different major, but it's too early to know.